source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.cpp

```diff
@@ -105,24 +105,23 @@ std::string CreateMaterialXNodeCmd::createNode( mx::DocumentPtr document,
         ogsFragment->getFragmentName(), ogsFragment->getFragmentSource()
     );
 
-    // Decide what kind of node we want to create
-    if (!createAsTexture)
```

Comment:
createAsTextture could be false either because the -t flag was omitted or because it was set to 0 which used to mean "force-create a surface node".